Sources/TSCUtility/Triple.swift

```diff
@@ -156,7 +160,23 @@ public struct Triple: Encodable, Equatable {
     }
 
     public func isDarwin() -> Bool {
-        return vendor == .apple || os == .macOS || os == .darwin
+        return vendor == .apple || os == .macOS || os == .darwin || os == .ios || os == .watchos || os == .tvos || os == .driverkit
```

Comment:
Ideally we wouldn't include DriverKit in isDarwin, but since vendor == .apple was already there, technically this change is a no-op. So, it's fine for now, but something we may want to adjust later.